behave_test_automation/pah_behave/features/steps/new_drive_steps.py

```diff
@@ -0,0 +1,29 @@
+from behave import then, when
+from behave.matchers import use_step_matcher
+
+use_step_matcher("re")
+
+
+@when(
+    'User submits "([^"]*)", "([^"]*)", "([^"]*)" and "([^"]*)"')
+def submit_add_new_drive_required_fields(context, start_location, starting_mileage,
+                                         end_location, end_mileage) -> None:
+    context.add_new_drive_page.submit_add_new_drive_required_fields(start_location,
+                                                                    starting_mileage,
+                                                                    end_location,
+                                                                    end_mileage)
+
```

Comment:
```suggestion
    context.add_new_drive_page.submit_add_new_drive_required_fields(
        start_location,
        starting_mileage,
        end_location,
        end_mileage
)
```

So the problem for me here is that we don't have an agreed standard of coding, which comes in handy when the decisions on formatting might differ between programmers. I'd write this piece of code like that (^ suggestion), but because we don't have any standard - your way is also okay.

Reply:
Changes applied